lib/assessment.rb

```diff
@@ -20,20 +30,35 @@
 # Name and location should be write-only, but age should be read-only.
 
 ##
-# your code here
+class Person
+  attr_writer: name
+  attr_writer: location
+  attr_reader: age
+
+  def initialize(name,age,location)
+    @name = name
+    @age = age
+    @location = location
+  end
+end
 ##
 
 # Question 4
 # Create a new Person object using parameters "Dave", 32, and "Ohio".
 # Then, set that Person's location to "Somerville"
 
-Answer.dave = nil # replace nil with your code
-Answer.dave.location = nil # replace nil with your code
+Answer.dave = Person.new("Dave", 32, "Ohio")
+Answer.location = Answer.dave.location = "Somerville" # replace nil with your code
```

Comment:
Hint: You shouldn't have done this part `Answer.location = Answer.dave.location`